packages/vue-app/template/App.js

```diff
@@ -295,18 +295,37 @@ export default {
     <% } /* splitChunks.layouts */ %>
     <% } /* features.layouts */ %>
     <% if (isFullStatic) { %>
+    getRouterBase() {
+      return (this.$router.options.base || '').replace(/\/+$/, '')
+    },
+    getRoutePath(route = '/') {
+      const base = this.getRouterBase()
+      if (base && route.startsWith(base)) {
+        route = route.substr(base.length)
+      }
+      return (route.replace(/\/+$/, '') || '/').split('?')[0].split('#')[0]
+    },
+    getStaticAssetsPath(route = '/') {
+      const { staticAssetsBase } = window.<%= globals.context %>
+
+      return urlJoin(this.getRouterBase(), staticAssetsBase, this.getRoutePath(route))
+    },
+    async fetchStaticManifest() {
+      console.log(urlJoin(this.getStaticAssetsPath(), 'manifest.js'))
+      return window.__NUXT_IMPORT__('manifest.js', encodeURI(urlJoin(this.getStaticAssetsPath(), 'manifest.js')))
```

Comment:
Even if unnecessary, i would create `nuxtImport` method to wrap `window.__NUXT_IMPORT__` call

Reply:
I think it could be move if used elsewhere (and I think could be used more as a module util)

Reply:
jsonp is mostly an internal API...